frontend-client/src/components/HeaderTeam.vue

```diff
@@ -123,7 +123,7 @@ export default {
 }
 
 .btn-magenta-header:hover {
-  box-shadow: 0 12px 16px 0 rgba(0,0,0,0.24),0 17px 50px 0 rgba(0,0,0,0.19) !important;
+  box-shadow: 0 12px 16px 0 var(--main-magenta-color) ,0 17px 50px 0 rgba(0,0,0,0.80) !important;
```

Comment:
I think the offset can be reduced, because it "shines" into the the text field below and the transition could be a bit faster to make it feel snappier

Reply:
Thanks, it is way snappier with the reduced transition time and I also reduced the vertical offset so it doesn't overflow.

Reply:
For me it does still overflow, the new transition time looks great though! <img width="288" alt="Bildschirmfoto 2020-05-30 um 12 20 19" src="https://user-images.githubusercontent.com/4086170/83325863-f1cc7780-a26f-11ea-8a5e-cb33635c127b.png">